src/phpdoc/type_parser.go

```diff
@@ -63,6 +71,12 @@ const (
 	// Args[1] - val expression (right)
 	ExprKeyVal
 
+	// ExprMemberType is access to member.
+	// Examples: `\Foo::SOME_CONSTANT` `\Foo::$a`
+	// Args[0] - class fqn expression (left)
+	// Args[1] - member name expression (right)
```

Comment:
Can we store both `Args[0]` and `Args[1]` as ExprName?
I'm 100% positive that we can store `Args[0]` as ExprName, but not so sure about the `Args[1]`.
Maybe it's OK to have `ExprMember` for now, until we start using it and then it will be more apparent what do we want to do with it.